src/mantine-hooks/src/use-timeout/use-timeout.ts

```diff
@@ -1,16 +1,16 @@
 import { useRef, useEffect } from 'react';
 
 export function useTimeout(
-  fn: () => void,
```

Comment:
Why fn was renamed to callback?

Reply:
It was called callback in docs so I thought I change it to be consistant. callback seemed more descriptive than fn to me.